Add created and modified fields to user model

rnegron opened the pull request:
Instead of adding one-off fields, I thought it would be nice to install the [django-model-utils](https://github.com/jazzband/django-model-utils) package and use some of its model fields now that the codebase supports it.

Closes #95.

rnegron commented:
Just realized that this will add the `datetime` of when the migration is applied to all the user rows it updates by default. Probably not intended. Will fix.

jpadilla commented:
> Just realized that this will add the datetime of when the migration is applied to all the user rows it updates by default. Probably not intended. Will fix.

This is not a huge deal. If we wanted to get closer to approx. we could use a user's first comment/review's created date.

rnegron commented:
I went ahead and implemented your idea of using an existing user's first comment. I added the code into the migration file. Also removed `django_model_utils` since it was giving me trouble while running custom code in the migration

jpadilla commented:
@rnegron nice! could you check why tests are failing?

rnegron commented:
Oops, didn't mean to dismiss the review. Anyway, it should be good now!

jpadilla commented on notaso/users/fixtures/dummy.json:
```diff
@@ -2,12 +2,14 @@
   "pk": 1,
   "fields": {
-      "password": "pbkdf2_sha256$36000$nQmi1RHXkxtM$wLJEFsVaXDQ0uxEDsR30j72A4N8g0sHcXPLTvdlz4DM=",
+      "password": "password",
```
@rnegron any reason for changing this?

jpadilla commented:
@rnegron just deployed and ran migrations, all good. Thanks!
